package.json

```diff
@@ -17,13 +18,15 @@
     "@types/aws-lambda": "^8.10.125",
     "@types/lambda-log": "^3.0.2",
     "esbuild": "^0.19.5",
+    "esbuild-visualizer": "^0.4.1",
     "serverless": "^3.36.0",
     "serverless-better-credentials": "^2.0.0",
     "ts-node": "^10.9.1",
     "typescript": "^5.2.2"
   },
   "dependencies": {
     "aws-lambda": "^1.0.7",
-    "lambda-log": "^3.1.0"
+    "lambda-log": "^3.1.0",
+    "open": "^8.4.2"
```

Comment:
I believe this should be a dev dependency

Reply:
It can be yes, I mainly opened it to show what is possible. From the blog article, it seemed that you don't want to ship the sourcemaps with? So my idea with this PR wasn't that it will get merged, just to show that it is possible. Else if merged you would have to change parts of the article? 

If not and you do want to merge it, by all means :). Please feel free to make the small change above or I can as well. 

Nice article btw :). Also FYI, I created two issues that will/might turn into PRs upstream on the CDK:
- https://github.com/aws/aws-cdk/issues/28087
- https://github.com/aws/aws-cdk/issues/28088